Alarmi/Alarmi/Sources/ViewControllers/TodayViewController.swift

```diff
@@ -10,12 +10,108 @@ import UIKit
 
 class TodayViewController: UIViewController {
 
+    // MARK: View
+    private let stackView: UIStackView = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.axis = .vertical
+        $0.distribution = .fill
+        $0.alignment = .leading
+        $0.spacing = 8
+        return $0
+    }(UIStackView())
+
+    private let descriptionLabel: UILabel = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.text = "마지막으로 전화한지 5일이 지났어요."
+        $0.font = UIFont.preferredFont(forTextStyle: .title2)
+        return $0
+    }(UILabel())
+
+    private let subDescriptionLabel: UILabel = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.text = "2일 후면 전화하기로 한 날이에요."
+        $0.font = UIFont.preferredFont(forTextStyle: .body)
+        $0.textColor = .secondaryLabel
+        return $0
+    }(UILabel())
+
+    private let delayButton: UIButton = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+
+        let config = UIImage.SymbolConfiguration(font: UIFont.preferredFont(forTextStyle: .caption2))
+        let image = UIImage(systemName: "chevron.right", withConfiguration: config)
+        $0.configuration?.image = image
+        $0.configuration?.imagePlacement = .trailing
+        $0.configuration?.baseBackgroundColor = .systemBlue
+        $0.configuration?.titleAlignment = .leading
+        $0.configuration?.contentInsets = .zero
+
+        var container = AttributeContainer()
+        container.font = UIFont.preferredFont(forTextStyle: .footnote)
+        $0.configuration?.attributedTitle = AttributedString("이번 연락 미루기", attributes: container)
+
+        return $0
+    }(UIButton(configuration: .plain()))
+
+    private let callButton: UIButton = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.configuration?.baseBackgroundColor = .systemBlue
+        $0.configuration?.titleAlignment = .center
+        $0.configuration?.cornerStyle = .medium
```

Comment:
p2;

Dynamics Type이 적용되어 버튼 글자 크기가 달라지거나, 추후에 라이팅 수정으로 인해 버튼 문구가 달라질 수 있으므로 확장성을 위해 여기서 글자에 패딩값을 주어 버튼 크기가 변할 수 있도록 해주세요.

패딩값은 추후 디자인이 수정될 수 있으니 일단 아무값이나 넣으셔도 됩니다.
```suggestion
        $0.configuration?.cornerStyle = .medium
        $0.configuration?.contentInsets = NSDirectionalEdgeInsets.init(top: 16, leading: 16, bottom: 16, trailing: 16)
```

Reply:
확인했습니다.